style.css

```diff
@@ -68,23 +71,22 @@ a {
    margin-top: 1.8rem;
    height: 1.5rem !important;
    width: auto !important;
-   /*border-left: 1px solid #dfdfdf;*/
-   padding: 2rem
-   background-color: #8B8989;
+   padding-right: 1rem;
 }
 
 #sidebar h2 {
    font-size: 1.5rem;
 }
 
 #sidebar {
+   opacity: 1;
    background-color: #FFFFAA;
    color: #555555;
    padding: 2rem 0;
    overflow: none;
    font-size: 1.25rem;
    width: 37rem !important;
```

Comment:
Ha! got me. these tricks don't work on you, hey. Not to trick you, though. Ok...cool, I'll fix that right away